### PR TITLE
Fix coding style for mac implement

### PR DESCRIPTION
**Pull request by lisa0314**

### CODE REVIEW

**Comment by lisa0314** [comment]
Fix [otcshare/webml-polyfill#92](https://github.com/otcshare/webml-polyfill/issues/92)
@fujunwei  Please help to review.

File: services/ml/compilation_impl_mac.h [hunk 1]
```diff
@@ -55,35 +54,48 @@ struct OperationMac : public Operation {
...
+                                 bool depthwise = false);
+  bool CompileConv2DBNNS(OperationMac&);
   bool CompileConv2DOrDepthwiseConv2D(OperationMac&);
+  bool CompileAverageOrMaxPool2DBNNS(OperationMac&);
   bool CompileAverageOrMaxPool2D(OperationMac&);
+  bool CompileSoftmaxBNNS(OperationMac&);
   bool CompileSoftmax(OperationMac&);
+  bool CompileReshapeBNNS(OperationMac&);
   bool CompileReshape(OperationMac&);
+  bool CompileConcatenationBNNS(OperationMac&);
```
[the remark this answers is not in the document]
Comment (lisa0314) [reply services/ml/compilation_impl_mac.h]: The functions in compilation_impl_mac.mm will need to keep the same order? @fujunwei

[the remark this answers is not in the document]
Comment (fujunwei) [reply services/ml/compilation_impl_mac.h]: Yes

**Comment by lisa0314** [comment]
@fujunwei , Thanks for your review. Done.

**Comment by fujunwei** [comment]
LGTM.

**Pull request merged**